tests/playbooks/tests_802_1x.yml

```diff
@@ -3,6 +3,18 @@
 - hosts: all
   vars:
     interface: 802-1x-test
+    # Fixed versions/NVRs:
+    # 1.25.2
+    # NetworkManager-1.24.2-1.fc33
+    # NetworkManager-1.22.14-1.fc32
+    # NetworkManager-1.20.12-1.fc31
+    # 1.18.8
+    __NM_capath_ignored_NVRs:
```

Comment:
Should this be prefixed with __network too?

Reply:
I considered this but then it becomes harder to keep the line length. Since it is a variable used in a test and not exposed directly from the role, I don't think it needs to use the prefix.